internal/operators/ocs/validations.go

```diff
@@ -196,6 +195,21 @@ func gbToBytes(gb int64) int64 {
 	return gb * int64(units.GB)
 }
 
+// count all disks of drive type ssd or hdd
+func getDisks(disks []*models.Disk) int {
+
+	var countDisks int
+
+	for _, disk := range disks {
+		if disk.DriveType == ssdDrive || disk.DriveType == hddDrive {
```

Comment:
No size validation?  I think Orit said at least 10GB?

Reply:
We agreed on not to restrict any disk size. @nbalacha  to confirm.

Reply:
Correct - we are not validating the size of the OSD.

Reply:
@avishayt , will the `DriveType` always be  `SSD, HDD` or it can also be `ssd, hdd (lowercase)`?

Reply:
All caps, as you can see here:
https://github.com/openshift/assisted-service/blob/master/internal/hardware/validator.go#L90